js/ui/scheduler/rendering_strategies/ui.scheduler.appointments.strategy.base.js

```diff
@@ -1,36 +1,41 @@
-var noop = require("../../../core/utils/common").noop,
-    Class = require("../../../core/class"),
-    extend = require("../../../core/utils/extend").extend,
-    errors = require("../../widget/ui.errors"),
-    dateUtils = require("../../../core/utils/date"),
-    isNumeric = require("../../../core/utils/type").isNumeric,
-    typeUtils = require("../../../core/utils/type"),
-    themes = require("../../themes");
+import BasePositioningStrategy from "./ui.scheduler.appointmentsPositioning.strategy.base";
```

Comment:
This name ("base") was used because it is consistent with rendering strategies. Rendering strategies are BaseRenderingStrategy, HorizontalRenderingStrategy, VerticalRenderingStrategy and so on.